src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -61,16 +61,14 @@ private constructor(
               if (acc.lastOrNull()?.speaker == msg.speaker) acc else acc.plus(msg)
             }
 
-    val modelFromState = model?.let { ChatModel.fromDisplayNameNullable(it) }
-
     val selectedItem =
         chatPanel.modelDropdown.selectedItem?.let {
           ChatModel.fromDisplayNameNullable(it.toString())
         }
     val restoreParams =
         ChatRestoreParams(
             //        TODO: Change in the agent handling chat restore with null model
```

Comment:
Can we remove this Todo now?

Reply:
No. We need to pass `null` instead of `DEFAULT_MODEL.agentName` here.

Reply:
Not yet